blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+way, if you haven't joined our [Matrix Dasharo space](https://matrix.to/#/#dasharo:matrix.org)
+yet, do it quick! That is where all interesting stuff happens, like
+various technical discussions, hardware setups, firmware configurations and
+encountered issues.
+
+Let's have a look at the changelog.
+
+### Added
+
+- [Early boot DMA protection menu option](https://github.com/Dasharo/dasharo-issues/issues/275)
+
+  Many people reported various issues when DMA protection was introduced in
+  v1.1.0. Apparently certain GPU cards contain broken OptionROMs which cause
+  DMA errors and display initialization problems. Thus the firmware setup will
+  now contain an option to enable/disable the DMA protection on-demand (with
+  default state being disabled to avoid problems with dGPUs). Additionally an
+  option to keep IOMMU enabled during OS control hand-off (actually during the
+  UEFI Exit Boot Services) has been added. This setting may be especially
+  interesting for people seeking for the most secure configuration of the
+  firmware. But keep in mind, your Windows will not boot if you enable this
+  option! Refer to our [firmware setup documentation](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+  for more details.
+
+- ACPI PCI interrupt routing for CPU PCIe Root Ports
+
+  The CPU PCI Express root ports had no interrupt routing information reported
+  in ACPI. Just a small fix.
+
+- OC Watchdog ACPI device as in MSI firmware
+
+  Original MSI firmware has been reporting an ACPI device describing the
+  overclocking watchdog. This watchdog is being used by overclocking platforms
+  since 6th generation of Intel Core processors. The same ACPI device has been
+  added to Dasharo to keep consistency. Possibly Windows overclocking tools may
+  depend on it.
+
+### Changed
+
+- Updated SMMSTORE driver to upstream version in UEFI Payload
+
+  An update to the driver serving as a backend to write variables to the SPI
+  flash from UEFI Payload. The change was introduced due to problems with
+  variable updates on other Dasharo platform. The advantage of the update is a
+  cleaned up code with better quality.
+
+- [Improved visual comfort in the boot manager](https://github.com/Dasharo/dasharo-issues/issues/286)
+
+  Minor visual improvement reported by one of laptop users unable to
+  distinguish the entries in the Boot Manager boot device selection window. Now
+  the currently selected device is indicated by an arrow and highlighted with
+  blue color.
+
+### Fixed
+
+- [MSI PRO Z690-A WIFI DDR4 doesn't initialize IGP on certain Processors models](https://github.com/Dasharo/dasharo-issues/issues/274)
+
+  This fix is a crucial one in this release. Many users reported non-working
+  on-board displays with different CPUs than those which were tested in 3mdeb
+  lab. Now all Alder Lake-S graphics devices should be initialized properly and
+  give firmware output on the screen.
```

Comment:
maybe referring to HCL report here and encouraging sending HCL reports through DTS, but from Dasharo-ed devices make sense here?

Reply:
Sending an HCL would not help here. SOme people would have problems booting the DTS because of display not working

Reply:
@miczyg1 what help you in debugging those kind of issues, what stuff users can deliver to you to helped you solve above problems? Then when can be changed in users reports or what additional actions users can take to help you resolve future problems?

Please make this achievement actionable, we deliver something **crucial** and we would like in return get some activity from users. It has to be simple, it has to be linkable (to avoid DRY) it has to be something that potentially affected users can do easily and bring value to rest of Dasharo community.

Reply:
Added something that may be usable: https://github.com/3mdeb/news-and-ideas/pull/513/commits/8209b923dc9734cfd7749f6c4a71378bd1fe5f57

Reply:
There is typo in the text.

I think better sentence could be:

```
If you are unable to boot the DTS to provide [HCL report](https://docs.dasharo.com/dasharo-tools-suite/documentation/#hcl-report), then try using `fwdump-docker` image or `cbmem` from the [following guides](https://docs.dasharo.com/common-coreboot-docs/dumping_logs/).
```

Reply:
Applied